dateutil/test/test_parser.py

```diff
@@ -675,6 +675,17 @@ def test_era_trailing_year(self):
         res = parse(dstr)
         assert res.year == 2001, res
 
+    def test_first_century(self):
```

Comment:
Can you make this a parametrized test? I prefer "one assert per test" wherever possible.

Reply:
Will do.